Corgibytes.Freshli.Cli/Corgibytes.Freshli.Cli.csproj

```diff
@@ -4,10 +4,23 @@
     <TargetFramework>net5.0</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Corgibytes.Freshli.Lib" Version="0.0.4" />
+    <PackageReference Include="Corgibytes.Freshli.Lib" Version="0.4.0-alpha0208" />
```

Comment:
Will have to make a note we updated the CLI to v0.4.0-alpha0208.

Reply:
Is this a comment that I need to make an update in a readme somewhere? I believe I had to update the CLI to use that version of Lib because only the alpha was available to me on githubpackages or nuget.org